server/services/subscription.ts

```
@@ -2,6 +2,7 @@ import { Subscription } from "../models/subscription";
 import type { HemocioneUserAuthTokenData } from "./auth";
 import { getEventBySlug, incrementEventScheduleOccupiedSlots } from "./event";
 import { getCleanFullName } from "~/utils/getCleanFullName";
+import { getBrazilTodayStart, getBrazilTomorrowStart, getBrazilTomorrowEnd } from "../utils/brazilTimezone";
```

ESLint

[error] `../utils/brazilTimezone` import should occur before type import of `./auth`

(import/order)